djangobaseview/static/js/guestbook/view/GreetingView.js

```diff
@@ -0,0 +1,49 @@
+define([
+	"dojo/dom",
+	"dojo/on",
+	"dojo/dom-construct",
+	"dojo/_base/declare",
+	"./_ViewBaseMixin",
+	"../store/GreetingStore",
+	"dojo/text!../templates/GreetingView.html"
+], function (dom, on, domConstruct, declare, _ViewBaseMixin, GreetingStore, template) {
+
+	return declare("guestbook.view.GreetingView", [_ViewBaseMixin], {
+		templateString: template,
+
+		constructor: function(guestbook) {
+			this.greetingId = guestbook.greeting_id;
+			this.createDate = guestbook.date;
+			this.name = guestbook.guestbook_name;
+			this.content = guestbook.content;
+			this.updatedBy = guestbook.updated_by;
+			this.updatedDate = guestbook.updated_date;
+		},
+
+		_deleteGreeting: function () {
+			var confirm = window.confirm("Do you want delete " + this.name);
+			if(confirm == true) {
+				var store = new GreetingStore(this.name, this.content, this.greetingId);
+				var greetingViewId = this.id;
+				store.deleteGreeting().then(function() {
+					domConstruct.destroy(greetingViewId);
```

Comment:
- hãy dùng `this.destroy()`
  - https://dojotoolkit.org/reference-guide/1.10/dijit/_WidgetBase.html
- hãy dùng `lang.hitch()` để truyền objectThis vào
  - https://dojotoolkit.org/reference-guide/1.10/dojo/_base/lang.html#hitch